### PR TITLE
layers/+lang/python/funcs.el: Minor enhance for the doc and code format

### PR DESCRIPTION
**Pull request by sunlin7**
Minor change for the function document, and code re-format.

### CODE REVIEW

File: layers/+lang/python/funcs.el [hunk 1]
```diff
@@ -217,15 +217,14 @@ ROOT-DIR should be the path for the environemnt, `nil' for clean up"
 
 ;; from https://www.snip2code.com/Snippet/127022/Emacs-auto-remove-unused-import-statemen
 (defun spacemacs/python-remove-unused-imports ()
-  "Use Autoflake to remove unused function"
-  "autoflake --remove-all-unused-imports -i unused_imports.py"
+  "Use Autoflake to remove the unused imports:
```
Comment (lebensterben) [inline layers/+lang/python/funcs.el]: the first line of any function documentation should be a complete sentence ends with a period (.).

File: layers/+lang/python/funcs.el [hunk 2]
```diff
@@ -217,15 +217,14 @@ ROOT-DIR should be the path for the environemnt, `nil' for clean up"
...
-      (progn
-        (shell-command (format "autoflake --remove-all-unused-imports -i %s"
-                               (shell-quote-argument (buffer-file-name))))
-        (revert-buffer t t t))
-    (message "Error: Cannot find autoflake executable.")))
+  (cond ((executable-find "autoflake")
+         (shell-command (format "autoflake --remove-all-unused-imports -i %s"
+                                (shell-quote-argument (buffer-file-name))))
+         (revert-buffer t t t))
+        (t (message "Error: Cannot find autoflake executable."))))
```
Comment (lebensterben) [inline layers/+lang/python/funcs.el]:
```suggestion
        (t (user-error "Cannot find autoflake executable"))))
```

**Review by lebensterben — CHANGES_REQUESTED** [review]

File: layers/+lang/python/funcs.el (hunk 1 above)
Comment (sunlin7) [reply layers/+lang/python/funcs.el]: Modified in that latest change.

File: layers/+lang/python/funcs.el (hunk 2 above)
Comment (sunlin7) [reply layers/+lang/python/funcs.el]: Good idea. And included in the latest commit. Thanks.

**Comment by sunlin7** [comment]
@lebensterben 
I had modified the changes with your comments. 
Please help review again. Thanks.

**Comment by lebensterben** [comment]
Hi I've changed a little bit to deal with non-zero exit code from autoflake.
If that looks good to you, I will merge this PR.

(ignore they other changed files as they are from other commits)

**Comment by sunlin7** [comment]
Hi @lebensterben 
I had check the change line by line, it's okay for me. Thanks !

**Pull request merged**